contract/tests/test_contract.py

```diff
@@ -669,14 +669,17 @@ def test_act_show_contract(self):
         show_contract = self.partner.with_context(
             contract_type="sale"
         ).act_show_contract()
-        self.assertDictContainsSubset(
```

Comment:
This is done on purpose and you shouldn't change it.

Reply:
This change is to avoid warning log (deprecated). Change is the same as the one currently in 15.0 (https://github.com/OCA/contract/blob/15.0/contract/tests/test_contract.py#L637-L649)

Reply:
But this is changing the meaning of the test. This can be done if you are sure that the dictionary is always going to return the same, but imaging a module that adds a new field in the act_window model, and if you do `read`, that value will be included in your return, not matching exactly the one expected, but being still correct.

Reply:
Ok, this change is removed.

Reply:
What do you think about using `assertLessEqual`?